daml-lf/spec/daml-lf-1.rst

```diff
@@ -298,6 +298,11 @@ Version: 1.dev
 
   + **Add** generic map type ``GenMap``.
 
+  + **Change** the operational semantics to make the evaluation order of
+    sub-expressions, and memoization of values, implementation-defined.
+    This change preserves the determinism of ``Ok`` results, but makes
+    ``Err`` results non-deterministic.
+
```

Comment:
The comment should not be there. This log tracks semantics changes or serialization changes by version. Here we did not change the `dev` version of LF, but "fix" a bug in the spec for all version of LF.
We usually do not track bug fixes in the specification, I will do the same for this change.